### DISCUSSION TITLE
左滑退格键删掉一段文本的问题

### DISCUSSION THREAD

**Discussion by SolidBlock-cn** [fcitx5-android/fcitx5-android#549] [Q&A | 问答] [answered]
当候选栏中有上一个词语的联想时，左划退格键方便地删除一段文本时，不会清除联想，且会重置键盘中的引号（“”）的左右状态。

**Comment by rocka** [comment, marked as answer]
左划退格清除联想在 https://github.com/fcitx5-android/fcitx5-android/commit/c638e408008a06eaa54e4e90b521c4eb6404f626 已经支持了，请更新到 GitHub Release 或者 F-Droid 最新版本

重置引号状态这个大概没办法
